### PR TITLE
nfbc: Add version 1.6.3

### PR DESCRIPTION
**Pull request by r0uv3n**
Hi, first contribution. Hope full name in description and no autoupdate on the hash are the right ways to do things (hash has to be generated from downloaded file). This uses Expand-DarkArchive as done for example in the [portable vcreddist 2013 manifest](https://github.com/lukesampson/scoop-extras/blob/a8bcb1b43b0230a9348a422017be31fe625cea70/bucket/vcredist2013-portable.json)).

### CODE REVIEW

File: bucket/nbfc.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "version": "1.6.3",
+  "description": "NoteBook FanControl, a cross-platform fan control service for notebooks",
+  "homepage": "https://github.com/hirschmann/nbfc/",
+  "checkver": "github",
+  "license": {
+    "identifier": "GPL-3.0-only"
+  },
```
Comment (Ash258) [inline bucket/nbfc.json:8]: license: GPL-3.0-only

File: bucket/nbfc.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "hash": "5A42531F3A051DBD9664B1FF41A6CDBFBB34BC71A91DB719B8FE94829594EBAA",
+  "url": "https://github.com/hirschmann/nbfc/releases/download/1.6.3/NoteBookFanControl.1.6.3.setup.exe#/setup.exe",
```
Comment (Ash258) [inline bucket/nbfc.json:10]: renaming exe to exe does not make sense

File: bucket/nbfc.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  "pre_install": [
+    "Get-ChildItem \"$dir\\setup.exe\" | ForEach-Object { Expand-DarkArchive $_ \"$dir\\nbfc\" -Removal }",
```
Comment (Ash258) [inline bucket/nbfc.json:12]: $fname

**Review by Ash258 — CHANGES_REQUESTED** [review]
./bin/checkver.ps1 nbfc -f

**Comment by r0uv3n** [comment]
I just noticed I forgot to register the nbfc service, this pull request is definetely incomplete, will be working on this.

**Comment by r0uv3n** [comment]
It seems I'm in over my head here, so I should probably close this.

**Pull request closed without merging**